src/index.js

```diff
@@ -1,36 +1,69 @@
+/* eslint-disable import/no-cycle */
+import checkItem from './checkItem.js';
+import descriptionOnFocus from './inputsFocus.js';
+import {
+  onMouseOver, onMouseOut, onDragStart, onDragEnd, onDragOver,
+} from './dragOverAndDrop.js';
+import {
+  addNewItem, clearAll, updateInput, removeItem,
+} from './addRemove.js';
 import './index.css';
 
-const Tasks = [
-
-  {
-    description: 'Do the list',
-    completion: false,
-    index: 0,
-  },
-  {
-    description: 'Study hard before the class',
-    completion: false,
-    index: 1,
-  },
-  {
-    description: 'Do the project',
-    completion: false,
-    index: 2,
-  },
-];
-
-const workList = document.querySelector('.todoList');
-
-function chore() {
-  Tasks.forEach((task) => {
-    workList.innerHTML += `<li class="item-list" data-key="${task.index}">
-          <div class="checkList">
-              <input type="checkbox" name="checkbox" id="checkbox" data-key="${task.index}">
-              <input type="text" id="inputList"  value="${task.description}">
-              <button data-key="${task.index}" class="move-item"><i class="fas fa-ellipsis-v"></i></button>
-          </div>
-      </li>`;
+// SET TODO ARR
+let toDoArr = [];
+export const setToDoArr = () => {
+  if (JSON.parse(localStorage.getItem('myToDos'))) {
+    toDoArr = JSON.parse(localStorage.getItem('myToDos'));
+  } else {
+    toDoArr = [];
+  }
+};
+setToDoArr();
+const listWrap = document.querySelector('.todo-list-wrap');
+// IMPLEMENT TO DO ITEMS TO SCREEN
+export const implementToDoItems = (arr) => {
+  arr.forEach((item) => {
+    let isCompleted = '';
+    let completedClass = '';
+    if (item.completed) {
+      isCompleted = 'checked';
+      completedClass = 'completed-item';
+    }
+    listWrap.innerHTML += `<li class="list-item ${completedClass}" data-key="${item.index}" draggable="true">
+      <div class="list-item-check-name">
+        <input type="checkbox" class="check-item" data-key="${item.index}" ${isCompleted}>
+  <input class="description-input" data-key="${item.index}" type="text" value="${item.description}">
+      </div>
+      <button type="button" data-key="${item.index}" class="move-item"><i class="fas fa-ellipsis-v"></i></button>
+      <button type="button" data-key="${item.index}" class="remove-item"><i class="far fa-trash-alt"></i></button>
+    </li>`;
   });
-}
-
-chore();
+};
+implementToDoItems(toDoArr);
+// CHECK ITEMS
+const toDoChecksInputs = [...document.querySelectorAll('.list-item .check-item')];
```

Comment:
All variable definitions should be done at the beginning of the file as it helps readability